src/index.html

```diff
@@ -60,9 +62,6 @@ <h3>Cloud Data Warehouse Interoperability</h3>
       <p>Snowflake, BigQuery, RedShift, DataBricks can all work together seamlessly with the same geospatial data format</p>
```

Comment:
Would we want to add something like "and others" (after DataBricks) to not let it seem it's only about those 4 specific ones?